config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
why does this file differ so much from the one that comes by default? https://github.com/ankane/strong_migrations/blob/master/lib/generators/strong_migrations/templates/initializer.rb.tt